starter-dev-e2e/cypress/e2e/profile.cy.ts

```diff
@@ -130,3 +129,95 @@ describe("Profile Page", () => {
       .should("be.visible");
   });
 });
+
+describe("When there is proper empty profile page responses", () => {
+  beforeEach(() => {
+    cy.interceptGraphQLCalls(View.Profile, InterceptResponse.Empty);
+    cy.interceptRestCalls(View.Profile, InterceptResponse.Empty);
+
+    cy.visit("/thisdot");
+  });
+
+  it("should display correct name and username with empty information", () => {
+    cy.get(`[data-testid="profile page name"]`)
+      .should("be.visible")
+      .and("contain.text", "thisdotlabs")
+      .get(`[data-testid="profile page username"]`)
+      .should("be.visible")
+      .and("contain.text", "thisdot")
+      .get(`[data-testid="profile followers count"]`)
+      .contains("0")
+      .get(`[data-testid="profile following count"]`)
+      .contains("0")
+      .get(`[data-testid="profile starred count"]`)
+      .contains("0");
+  });
+
+  it("should have no organizations listed in profile", () => {
+    cy.get(`[data-testid="profile page orgs"]`).should("not.exist");
+  });
+
+  it("should not display any repos or be able to paginate previous and next ", () => {
+    cy.wait("@UserReposQuery")
+      .get(`[data-testid="profile repo list heading"]`)
+      .should("have.length", 0)
+      // next 4 lines does fails on angular-apollo due to disabled buttons
```

Comment:
Yeah this will fail in other tests too currently, we won't need this comment in the code now that I left this comment noting it 😁